test-project/Assets/Generated/Source/improbable/gdk/tests/ExhaustiveBlittableSingular.cs

```diff
@@ -95,6 +95,17 @@ public void MarkDataClean()
                 dirtyBits1 = 0x0;
             }
 
+            public Snapshot TakeSnapshot(global::Unity.Entities.World world)
```

Comment:
Rename this to ToComponentSnapshot I think.

Reply:
We could generally rename this to component snapshot. Could do that after this though.